server/swap/swap.go

```diff
@@ -1420,7 +1419,7 @@ func (s *Swapper) processAck(msg *msgjson.Message, acker *messageAcker) {
 	ack := new(msgjson.Acknowledgement)
 	err := msg.UnmarshalResult(ack)
 	if err != nil {
-		s.respondError(msg.ID, acker.user, msgjson.RPCParseError, "error parsing acknowledgment")
+		s.respondError(msg.ID, acker.user, msgjson.RPCParseError, fmt.Sprintf("error parsing acknowledgment: %v", err))
```

Comment:
Otherwise `err` is swallowed (updated a couple of similar places with the same issue, can find them all by searching `fmt.Sprintf` in this PR):
```
2022-07-27 11:57:10.586 [DBG] TEST: Error going to user 224cbaaafa80bf3bd1ff731590bcbdda5a76f91e60a156994634e91c0000002e, code: 1, msg: error parsing acknowledgment
```